docs/configuration.md

```diff
@@ -297,6 +302,18 @@ For more information on how to use puppeteer, read up on [their API docs](https:
 
 An object of options to pass to puppeteer's [`launch` method](https://github.com/puppeteer/puppeteer/blob/v2.0.0/docs/api.md#puppeteerlaunchoptions). Only used when `puppeterScript` is set.
 
+#### `psiApiKey`
+
+_method=psi only_
+
+The API key to use for making PageSpeed Insights requests. Required if using `method=psi`. You can obtain a PSI API key [from Google APIs](https://developers.google.com/speed/docs/insights/v5/get-started#key).
+
+#### `psiApiEndpoint`
```

Comment:
🔝👏

Reply:
Is just applause? :) 

Just want to make sure I'm not missing an action to be taken here